Cargo.toml

```diff
@@ -49,6 +49,9 @@ bytes  = "0.3.0"
 nix = "0.6.0"
 utime = "0.1"
 chrono = "0.2"
+prometheus = "0.2.2"
+lazy_static = "0.2.1"
+hyper = "0.9"
```

Comment:
As we discussed before, we don't want to add another port now. so now the best way is to logging metrics periodically, like 5m. 

/cc @ngaut